### DISCUSSION TITLE
Tabview broken since 4.4.2 and the introduction of lazyloading

### DISCUSSION THREAD

**Discussion by OrionDev23** [bdlukaa/fluent_ui#833] [Q&A] [answered]
Basically my code was working fine, and still is if i keep the version on 4.4.1

My issue is that whenever i create a new tab, or select another one, all Textboxes values or anything created on the body of others tabs is reset.

How can i fix this ?

(ps: i updated to the latest version (currently 4.6.0) and manually reverted back the tab_view.dart file like it was in 4.4.1 and it works fine after this
https://github.com/bdlukaa/fluent_ui/commit/00c61146cba44caab0577b69f1f7330cc4132775 )

**Comment by bdlukaa** [comment, marked as answer]
Seems related to #910. You can make use of `AutomaticKeepAliveClientMixin` in your widgets to save the state.
